website/addons/s3/templates/s3_node_settings.mako

```diff
@@ -1,104 +1,73 @@
-                    authorized by
-                    <a href="${owner_url}" target="_blank">
-                        ${owner}
+<div id="s3Scope" class="scripted">
+    <h4 class="addon-title">
+        Amazon S3            
+        <small class="authorized-by">
+            <span data-bind="if: nodeHasAuth">
+                authorized by <a data-bind="attr.href: urls().owner">
+                {{ownerName}}                    
+                </a>
+                % if not is_registration:
+                    <a data-bind="click: deauthorizeNode" class="text-danger pull-right addon-auth">
+                      Deauthorize
                     </a>
-                    % if not is_registration:
-                        <a id="s3RemoveToken" class="text-danger pull-right addon-auth">Deauthorize</a>
-                    % endif
-                % elif user_has_auth:
-                    <a id="s3ImportToken" class="text-primary pull-right addon-auth">Import Credentials</a>
                 % endif
-            </small>
-
-        </h4>
-    </div>
-
-    % if bucket_list is not None:
-
-        <div class="form-group">
-
-            <p> <strong>Current Bucket:</strong></p>
-
-            <div class="row">
-
-                <div class="col-md-6">
-
-                    <select class="form-control" id="s3_bucket" name="s3_bucket"
-                        ${'' if user_has_auth and user_is_owner and not is_registration else 'disabled'}>
-                        <option value="">-----</option>
-                        % for bucket_name in bucket_list or []:
-                            <option value="${bucket_name}" ${'selected' if bucket_name == bucket else ''}>
-                                ${bucket_name}
-                            </option>
-                        % endfor
-                    </select>
-
-                </div>
-
-                % if user_has_auth and user_is_owner and not is_registration:
-                    <div class="col-md-6">
-                        <a class="btn btn-default" id="newBucket">Create Bucket</a>
-
-                        <button class="btn btn-primary addon-settings-submit pull-right">
-                            Submit
-                        </button>
-                    </div>
-                % endif
-
-            </div>
-
-        </div> <!-- End form group -->
-
-    % elif node_has_auth and bucket_list is None:
-
-        <div>
-            <span class="text-danger">
-                Error loading S3 access keys. Please refresh the page.
             </span>
+            <span data-bind="if: showImport">
+                <a data-bind="click: importAuth" class="text-primary pull-right addon-auth">
+                  Import Access Token
+                </a>
+            </span>
+        </small>            
+    </h4>
+    <div data-bind="if: showSettings">
+        <p>
+            <strong>Current Bucket:</strong>
+            <span data-bind="text: currentBucket"></span>
```

Comment:
The other addons display the current selection as a link to the files page. Can you do the same here?